Sources/Sentry/SentryHub.m

```diff
@@ -262,6 +262,38 @@ - (void)captureCrashEvent:(SentryEvent *)event withScope:(SentryScope *)scope
     }
 }
 
+/**
+ * This method expects an abnormal session already stored to disk. For more info checkout: @c
+ * SentryCrashIntegrationSessionHandler
+ */
+- (void)captureFatalAppHangEvent:(SentryEvent *)event
+{
+    // We tread fatal app hang events similar to crashes.
+    event.isCrashEvent = YES;
```

Comment:
I will refactor this property in an upcoming PR. See https://github.com/getsentry/sentry-cocoa/issues/4935